src/main/resources/org/sagebionetworks/web/client/widget/discussion/ThreadWidgetViewImpl.ui.xml

```diff
@@ -2,14 +2,32 @@
 <ui:UiBinder xmlns:ui="urn:ui:com.google.gwt.uibinder"
 	xmlns:g="urn:import:com.google.gwt.user.client.ui" 
 	xmlns:b="urn:import:org.gwtbootstrap3.client.ui"
-	xmlns:bh="urn:import:org.gwtbootstrap3.client.ui.html">
+	xmlns:bh="urn:import:org.gwtbootstrap3.client.ui.html"
+	xmlns:t="urn:import:org.sagebionetworks.web.client.view.bootstrap.table">
 	<b:Row>
-		<bh:Div addStyleNames="margin-left-15 margin-right-15">
-			<bh:Paragraph>thread title</bh:Paragraph>
-			<bh:Paragraph>thread comment</bh:Paragraph>
-		</bh:Div>
 		<b:Column size="XS_12">
-			<bh:Div ui:field="replyListContainer"></bh:Div>
+			<t:Table ui:field="threadHeader" width="100%">
+				<t:TableRow>
+					<t:TableHeader width="20px" />
```

Comment:
remove these extra column.  when expanding, these should be in t:TableData rather than t:TableHeader.